.github/workflows/test.yml

```diff
@@ -0,0 +1,39 @@
+name: Lint & Test
+
+on:
+  push:
+  pull_request:
+    types: [reopened]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  # test:
+  #   name: Test
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@v3
+  #     - uses: actions/setup-node@v3
+  #       with:
+  #         node-version-file: .nvmrc
+  #         cache: yarn
+  #     - name: Install dependencies
+  #       run: yarn install --immutable
+  #     - name: Test
+  #       run: yarn test
```

Comment:
Just noticed another Stacy was tagged 😄 @stacyk

Reply:
@sanajaved7 this could explain why I was a bit lost when @jerivas said I probably got the notification. I pushed a fix for just these image paths, but that does't include testing for other broken links: https://github.com/oddbird/sass-site/commit/199bb0ea08e42fb6e302f70e53f4d7142cb006b2  I wish I had @ stacy  as a screen name on GH.